src/js/view/preview.js

```diff
@@ -1,13 +1,39 @@
 define([
-    'utils/underscore'
-], function(_) {
+    'utils/underscore',
+    'utils/helpers'
+], function(_, util) {
 
     var Preview = function(_model) {
         this.model = _model;
 
-        this.model.on('change:playlistItem', this.loadImage, this);
+        this.model.on('change:playlistItem', onPlaylistItem, this);
     };
 
+    function loadOnComplete() {
+        if (this.model.get('state') === 'complete') {
+            this.loadImage(this.model, this.model.get('playlistItem'));
+        }
+    }
```

Comment:
This would be better minified and written as:

```
    function onChangeState(model, state) {
        if (state === 'complete') {
            this.loadImage(model, model.get('playlistItem'));
        }
    }
```